sysbuild/CMakeLists.txt

```diff
@@ -490,6 +490,30 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_pre_cmake)
   if(SB_CONFIG_SDP)
     include_sdp()
   endif()
+
+  foreach(config APPROTECT_USE_UICR APPROTECT_LOCK APPROTECT_USER_HANDLING
```

Comment:
minor: Instead of looping on all the options of the choice, wouldn't it have been possible to use `SB_CONFIG_APPROTECT_HANDLING` directly? (I don't know)

Reply:
I don't think you can? At least not while the method of doing this is by set_config_bool.